content/en/blog/2025/infernece-extension-support/index.md

```diff
@@ -0,0 +1,155 @@
+---
+title: "Bringing AI-Aware Traffic Management to Istio: Gateway API Inference Extension Support"
+description: Introducing Gateway API Inference Extension support in Istio for optimizing traffic to AI Inference workloads.
```

Comment:
Description reads like the title re-written.  Maybe cut the "Gateway API.." part from the title?

Reply:
idk, I still think it sits nicely.

how about:

Description: Optimized routing for AI workloads in Istio, powered by the latest Gateway API Inference Extension.